### ISSUE TITLE
Invalid and hardcoded types-dashboard urls

### ISSUE THREAD

**Issue by FedeG** [lelylan/devices-dashboard-ng#91]
For example:

- http://lelylan.github.io/types-dashboard-ng/#/types/<i<id>d>/
![captura de pantalla de 2017-02-07 12-18-03](https://cloud.githubusercontent.com/assets/4097554/22697594/b1b417a4-ed30-11e6-9c44-3502ceaab506.png)

**Comment by FedeG** [comment]
This links:
```
app/bower_components/type-directive-ng/dist/views/templates/default.html:            <code><xmp id="embed"><iframe src="http://lelylan.github.io/types-dashboard-ng/#/types/{{type.id}}/embed" style="width:360px;height:440px;border:5px solid #f8f8f8;"></iframe></xmp></code>
app/bower_components/device-directive-ng/dist/views/templates/default.html:                <a class="ly-type-value" href="http://lelylan.github.io/types-dashboard-ng/#/types/{{type.id}}" target="blank">{{type.name}}</a>
dist/bower_components/type-directive-ng/dist/views/templates/default.html:            <code><xmp id="embed"><iframe src="http://lelylan.github.io/types-dashboard-ng/#/types/{{type.id}}/embed" style="width:360px;height:440px;border:5px solid #f8f8f8;"></iframe></xmp></code>
dist/bower_components/device-directive-ng/dist/views/templates/default.html:                <a class="ly-type-value" href="http://lelylan.github.io/types-dashboard-ng/#/types/{{type.id}}" target="blank">{{type.name}}</a>
```

Issue in device-directive-ng -> 
Issue in type-directive-ng ->

**Comment by andreareginato** [comment]
Can this issue be considered closed @FedeG?

**Comment by FedeG** [comment]
@andreareginato. Yes, this bug are fixed in #84
